documentation/Database-Schema-Application.md

```diff
@@ -0,0 +1,56 @@
+# Database Schema Application
+
+![Database Schema](images/database-schema-application.png)
+
+## Tables description: Application
+
+### Application instance
+The application has all the visualisation and logic defined in the <b>application template</b> [(described in Database Schema Template)](Database-Schema-Template.md). When the Applicant clicks to apply for a application type (aka template) a new instance of one application is created for them. The Applicant user can make changes until submitting the application to be reviewed.
+
+* <b>application</b>
+Representation of the application instance. All nested elements are accessible via joined tables and can be created or queried in the same call using the GraphQL engine.
+The `template_id` links to the template used to display all sections and elements (questions) to the user.
+The `user_id` links to the Applicant user that created this application.
+The `serial` is a number used to display the application number to users. 
+The `name` will be deduced by the template name + user + company names.
```

Comment:
Ah, okay!
We can do this in the front-end, but do you think it'd be better to write a postGres function to fill this field automatically?

Reply:
Yes, for sure! I'll start a document where we keep the possible useful functions to be added to postGres.